src/maplibre_helpers.ts

```diff
@@ -47,6 +47,25 @@ export function overwritePmtilesSource(map: Map, id: string, url: string) {
   });
 }
 
+// Like overwriteSource, but for geojson data hosted at a URL.
```

Comment:
See line 32. `overwriteSource` takes GJ data or a URL to it, because the underlying maplibre API handles it. We don't need this